src/Logic/operate.js

```diff
@@ -0,0 +1,39 @@
+import Big from 'big.js';
+
+const Operate = (number1, number2, operation) => {
+  if (number1 === 0) {
+    return number2;
+  }
+  if (number1 === 'Infinity') {
+    return 'Error';
+  }
+
+  let results = 0;
+  const numberOne = new Big(number1);
+  const numberTwo = new Big(number2);
+
+  switch (operation) {
+    case '-':
+      results = numberOne.minus(numberTwo);
+      break;
+    case '+':
+      results = numberOne.plus(numberTwo);
+      break;
+    case '*':
+      results = numberOne.times(numberTwo);
+      break;
+    case '/':
+      try {
+        results = numberOne.div(numberTwo);
+      } catch (error) {
+        results = 'Infinity';
+      }
+      break;
+    default:
+      results = numberTwo.div(100);
+      break;
+  }
+  return `${results}`;
+};
```

Comment:
The default state is meant to handle this case

Reply:
Thank you for your reply. It will be good to have it as one of the cases. This will allow you to use the default to return any other unexpected selection the user does.